test/unit/helpers/xhr/MFA_ENROLL_ACTIVATE_webauthn.js

```diff
@@ -0,0 +1,68 @@
+define({
+  "status": 200,
+  "responseType": "json",
+  "response": {
+    "stateToken": "testStateToken",
+    "expiresAt": "2018-08-17T02:38:18.000Z",
+    "status": "MFA_ENROLL_ACTIVATE",
+    "_embedded": {
+      "user": {
+        "id": "00ukjpMDb9O7CjJbx0g3",
+        "passwordChanged": "2018-08-15T18:31:57.000Z",
+        "profile": {
+          "login": "exampleUser@example.com",
+          "firstName": "Test",
+          "lastName": "User",
+          "locale": "en_US",
+          "timeZone": "America/Los_Angeles"
+        }
+      },
+      "factor": {
+        "id":"fuf52dhWPdJAbqiUU0g4",
+        "factorType":"webauthn",
+        "provider":"FIDO",
+        "vendorName":"FIDO",
+        "_embedded":{
+          "activation":{
+            "rp": {
```

Comment:
just for me to learn, what is `rp`?

Reply:
it stands for `webauthn relying party`: https://www.w3.org/TR/webauthn/#webauthn-relying-party

Reply:
got it. Why didn't we explicitly named it `relyingParty`?

Reply:
This is to be consistent with webauthn protocol, so that we can feed the entire `activation` object to browser's `navigator.credential.create` without making any changes.